integration/correctness/src/test/scala/ru/itclover/tsp/http/BasicJdbcTest.scala

```diff
@@ -1,143 +1,143 @@
-package ru.itclover.tsp.http
```

Comment:
Can we just remove those tests? Usually, commented code is never gonna be uncommented)

Reply:
The tests are temporarily disabled, so they are to be uncommented in the future)